core/src/main/java/it/pagopa/selfcare/mscore/core/NotificationServiceImpl.java

```diff
@@ -106,6 +106,28 @@ public void sendMailForApprove(User user, OnboardingRequest request, String toke
 
     }
 
+    public void sendMailForRegistrationNotificationApprove(User user, OnboardingRequest request, String token) {
+        Map<String, String> mailParameters;
+        mailParameters = mailParametersMapper.getOnboardingMailNotificationParameter(user, request, token);
```

Comment:
you might avoid one row....
`Map<String, String> mailParameters = mailParametersMapper.getOnboardingMailNotificationParameter(user, request, token);`